app/build.gradle.kts

```diff
@@ -17,7 +17,7 @@ android {
 
     defaultConfig {
         applicationId = "com.dd3boh.outertune"
-        minSdk = 24
+        minSdk = 27
```

Comment:
Is the min sdk bump and disabling build variants needed?

Reply:
I got this error because of the KizzyRPC dependency. What should I go with? 

uses-sdk:minSdkVersion 24 cannot be smaller than version 27 declared in library [com.github.dead8309:KizzyRPC:1.0.71]

	Suggestion: use a compatible library with a minSdk of at most 24,
		or increase this project's minSdk version to at least 27,
		or use tools:overrideLibrary="com.my.kizzyrpc" to force usage (may lead to runtime failures)